### ISSUE TITLE
Relocate files and remove unneeded

### ISSUE THREAD

**Issue by ozin7** [ayrinmo/html_sandbox#1]
https://github.com/ayrinmo/html_sandbox/blob/master/css/css.html - I think this could be removed.
https://github.com/ayrinmo/html_sandbox/blob/master/css/doc.html - I think this should be in the same directory as index.html

**Comment by ayrinmo** [comment]
When you delete and move something, you need to push?
Ok, i will do anything)

**Comment by ozin7** [comment]
@ayrinmo  yes, when you remove file `git status` will show it as green which means it will be added to the next commit

**Comment by ayrinmo** [comment]
https://nicepage.com/ru/html-templates/preview/zernovye-vysshego-kachestva-75472?device=desktop


ср, 27 січ. 2021 о 09:06 Mykhailo Gurei ***@***.***> пише:

**Comment by ayrinmo** [comment]
це зразок


пт, 7 трав. 2021 о 13:55 Ярина Мосюк ***@***.***> пише:

**Comment by ayrinmo** [comment]
file:///D:/my%20first%20project/teaching/html/kava.html

а це я зробила


пт, 7 трав. 2021 о 13:55 Ярина Мосюк ***@***.***> пише:
